### DISCUSSION TITLE
API Token

### DISCUSSION THREAD

**Discussion by TonyS001** [filecoin-project/lotus#11091] [General Q&A] [answered]
I run a lotus node and want to use api. 
I run `lotus auth create-token --perm write` and replace the `token` file in `.lotus` folder. But when I invoke the api, it still call permission err.
```
curl -X POST \
     -H "Content-Type: application/json" \
     --data '{
       "jsonrpc":"2.0",
       "method":"eth_newFilter",
       "params":[{"address": "0x6b175474e89094c44da98b954eedeac495271d0f", "toBlock": "latest", "fromBlock": "latest"}],
       "id":1
     }' \
     http://127.0.0.1:1234/rpc/v1
```
`{"jsonrpc":"2.0","id":1,"error":{"code":1,"message":"missing permission to invoke 'EthNewFilter' (need 'write')"}}`

**Comment by TippyFlitsUK** [comment]
Can you please provide lotus logs showing the denied query request @TonyS001? Many thanks 🙏

**Comment by beck-8** [comment, marked as answer]
？
Your replacement of the token file has nothing to do with curl not carrying the Bearer Token.
So your problem is that there is no token with write permission at all
